setup.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name="plug",
```

Comment:
what's plug needed for? a comment maybe?

Reply:
plug is the python module in this codebase